metrics/web/lib/dashboard/presentation/state/project_metrics_notifier.dart

```diff
@@ -250,10 +250,17 @@ class ProjectMetricsNotifier extends PageNotifier {
     _projectGroupModels = projectGroupModels;
 
     _refreshProjectGroupDropdownItemViewModels();
+    _updatePageParameters();
   }
 
   @override
-  void handlePageParameters(PageParametersModel parameters) {}
+  void handlePageParameters(PageParametersModel parameters) {
+    if (parameters == null) return;
+
+    final pageParameters = parameters as DashboardPageParametersModel;
+
+    _setPageParameters(pageParameters);
```

Comment:
🔨 
Should we set the values from the page parameters to the corresponding fields of this page notifier?